project-p1-first-webpage-master/code/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Intro Olivia Grabher</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" type="text/css"/>
+  </head>
+  <nav>
+    <menu>
+            <a href="#about-me">About Me</a>
+            <a href="#what-i-love">What I Love</a>
+            <a href="#projects">Projects</a>
+            <a href="#get-to-know-me">Contact</a>
+    </menu>
+    </nav>
+<body>
+<header>
+  <img src="./olivia-header-bild.jpg" alt="photo of olivia" class="img-olivia">
+<!--wie definiere ich Klassen richtig? nachschauen-->
+  <div class="name-photo">
+    <h1>Olivia Grabher </h1>
+  </div>
+<div class="description-olivia">
+  <p>Content Manager, Online Marketer &<br>Music Lover</p>
+</div>
+</header>
+<section>  
+    <h1> <span id="about-me"></span>About Me</h1>
```

Comment:
We should only use the `h1` tag once per page and I think that should be your name!💥 
But the h1 is only used once for accessibility purposes.